PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.cpp

```diff
@@ -706,7 +706,12 @@ bool PluginsAdminDlg::initFromJson()
 #ifdef DEBUG // if not debug, then it's release
 	
 	// load from nppPluginList.json instead of nppPluginList.dll
+#	ifdef __MINGW32__
+	// https://github.com/notepad-plus-plus/nppPluginList/issues/454
```

Comment:
Although some projects do it, I don't think I've ever seen an issue linked to in the N++ source code.  Not sure if Don will like that or not.

Reply:
So far at least 4 prior issues have been memorialized with comments, in 3 different source languages (bash, XML and C++).

~~~sh-session
$ git grep -n "notepad-plus-plus/issues"

PowerEditor/Test/FunctionList/bash/unitTest:5:# https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9019
PowerEditor/installer/functionList/lua.xml:2:<!-- Copied from  https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4563 -->
PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp:4267:     // ref: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9595#issuecomment-789824579
scintilla/win32/ScintillaWin.cxx:1339:  // https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5671
~~~

You may still be right, of course; Notepad++ isn't famous for being consistent.

Reply:
Issue link should be in the commit comment, not in the source code.